### ISSUE TITLE
Dashed lines?

### ISSUE THREAD

**Issue by pkofod** [jonathanBieler/BlackBoxOptimizationBenchmarking.jl#1]
Could some of the lines be dashed? I'm finding it a bit hard to see if Optim.NelderMead is the first in all the figures. Also seems like it's shorting out quite quickly since it's the fastest of them all? Could that have something to do with the poor performance?

**Comment by jonathanBieler** [comment]
The plots are not ideal yes (specially if I had more optimizers), I should maybe do them with plotly to make them interactive.

I haven't looked at the results in details, but NelderMead is a local optimizer so I think it solves the Sphere and other functions with a single minimum very fast, but never manage to solve the more complicated functions. You can see that when using NelderMead with restarts (in red) you can do a bit better. Optim's gradient descent behave in the same way.

**Comment by pkofod** [comment]
Yes I didnt expect it to do well on the difficult problems I was just curious about its performance vs R Optim Neldermead :)

**Comment by jonathanBieler** [comment]
I will try to include some of Python algorithms to compare, it seems to work with PyCall.

**Comment by jonathanBieler** [comment]
@pkofod I've added Python's Neldermead and CMA-ES, and improved the plots a bit (I haven't found how to put the line style in the legend, but I ordered so it's clear which one is better). Better ?

**Comment by ChrisRackauckas** [comment]
Can the legend show which lines are dashed?

**Comment by jonathanBieler** [comment]
It doesn't seems there's an option for that in Gadfly, if I want to add more optimisers I should probably do something like [that](http://coco.lri.fr/COCOdoc/_images/examplefigure_all.png), otherwise it will quickly become unreadable.
